src/components/Tag/Tag.Style.js

```diff
@@ -0,0 +1,25 @@
+"use client"
+
+import styled from "styled-components"
+
+const categoriesColor = {
```

Comment:
FYI - Since we're gonna have a light/dark theme make sure these colors can be visually appealing in both modes

Reply:
Noted